mne_bids/write.py

```diff
@@ -819,6 +820,11 @@ def write_raw_bids(raw, bids_basename, output_path, events_data=None,
         inferred from the stim channel using `mne.find_events`.
     event_id : dict | None
         The event id dict used to create a 'trial_type' column in events.tsv
+    anonymize : bool | dict
+        Whether to anonymize data that could be used to identify the subject
+        (e.g. study date and time). Optionally, `daysback` and `keep_his` can
+        be provided in a dict as keyword arguments for
+        :func:`mne.io.anonymize_info`
```

Comment:
can you expose this in an example?

I think this is a good candidate example: https://mne.tools/mne-bids/auto_examples/convert_group_studies.html#sphx-glr-auto-examples-convert-group-studies-py